util/types/time.go

```diff
@@ -1354,7 +1356,7 @@ func extractSingleTimeValue(unit string, format string) (int64, int64, int64, go
 	return 0, 0, 0, 0, errors.Errorf("invalid singel timeunit - %s", unit)
 }
 
-// Format is `SS.FFFFFF`.
+// extractSecondMicrosecond extracts second microsecond from a string and its format is `SS.FFFFFF`.
```

Comment:
1. s/second microsecond/second and microsecond
2. s/and its format is/of which format is `SS.FFFFFF`